docs/static/security/es-security.asciidoc

```diff
@@ -5,8 +5,11 @@
 {es} {ref}/configuring-stack-security.html[security is on by default] starting in 8.0.
 {es} generates its own default self-signed Secure Sockets Layer (SSL) certificates at startup. 
 
-For an on-premise {es} cluster, {ls} must establish a secure connection using the self-signed SSL certificate before it can transfer data.  
-On the other hand, {ess} uses standard publicly trusted certificates, and therefore setting a cacert is not necessary.
+{ls} must establish a Secure Sockets Layer (SSL) connection before it can transfer data to an on-premise {es} cluster. 
+{ls} must have a copy of the {es} CA that signed the cluster's certificates.
+When a new {es} cluster is started up _without_ dedicated certificates, it generates its own default self-signed Certificate Authority at startup.
```

Comment:
Already linking to this document in Line 5. Is the additional link warranted?

Reply:
A single link is likely best, but this line's presence is more useful/accurate than line 6 (since most security-conscious users will be providing their own PKI which will cause ES to _not_ generate self-signed certificate authority).